docs/user/dql/functions.rst

```diff
@@ -209,9 +209,24 @@ CBRT
 Description
 >>>>>>>>>>>
 
-Specifications:
+Usage: CBRT(number) calculates the cube root of a number
+
+Argument type: INTEGER/LONG/FLOAT/DOUBLE
+
+Return type: DOUBLE
 
-1. CBRT(NUMBER T) -> T
+(Non-negative) INTEGER/LONG/FLOAT/DOUBLE -> DOUBLE
+(Negative) INTEGER/LONG/FLOAT/DOUBLE -> DOUBLE
```

Comment:
I think you can combine this into a single line

Reply:
@margarit-h Any change needed for this or we can just merge it for now?

Reply:
I think, we can merge, I will combine this description into one line in my next PR, which is adding PPL support for CBRT().